mlir/tools/rocmlir-lib/CMakeLists.txt

```diff
@@ -76,9 +83,18 @@ if(BUILD_FAT_LIBROCKCOMPILER)
     ${LIBRARY_NAME}
   )
 
+  rocm_package_setup_component(lib${LIBRARY_NAME}
+        LIBRARY_NAME lib${LIBRARY_NAME}
+        PACKAGE_NAME ${PACKAGE_NAME}
+  )
+
+  rocm_install(FILES LICENSE
+    DESTINATION share/doc/${PACKAGE_NAME}
+    COMPONENT lib${LIBRARY_NAME})
+
   # Install Miir.h to ${CMAKE_INSTALL_PREFIX}/include/${PACKAGE_NAME}/
   # as part of component libroCompiler
-  install(FILES Miir.h
+  rocm_install(FILES Miir.h
     DESTINATION include/${PACKAGE_NAME}
     COMPONENT lib${LIBRARY_NAME})
```

Comment:
Why is this being installed to this component rather than the MLSE standard packaging?

Reply:
I'm not the right person to answer this question. I'd guess probably we wanted to deliver this package separately? @jerryyin   might common on this when he's back.

Reply:
@lawruble13 Could you clarify what the question means? What is the typical practice for `MLSE standard packaging` to install a header file that client will consume??